tests/js/spec/components/quickTrace.spec.jsx

```diff
@@ -0,0 +1,368 @@
+import React from 'react';
```

Comment:
If this is a new file, you could convert this to tsx. We can start writing new tests in typescript now. We likely won't be rewriting existing tests into ts yet.

Reply:
Sure, let me do that 👍

Reply:
Sure, let me do that right now 👍